flink-streaming-connectors/flink-connector-elasticsearch2/src/main/java/org/apache/flink/streaming/connectors/elasticsearch2/ElasticsearchSink.java

```diff
@@ -166,30 +265,37 @@ public void open(Configuration configuration) {
 			transportClient.addTransportAddress(transport);
 		}
 
-		// verify that we actually are connected to a cluster
+		return transportClient;
+	}
+
+	private boolean checkConnectionStatus(TransportClient transportClient) {
```

Comment:
`isConnected(TransportClient)` is a more suitable name for this method. Also, can simply return as
`return ImmutableList.copyOf(transportClient.connectedNodes()).isEmpty()`